.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_Performance_Tests.json

```diff
@@ -77,7 +77,7 @@
           ],
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT mean(\"value\") FROM \"wordcount_py27_results\" WHERE metric = 'Python performance test' AND $timeFilter GROUP BY time($__interval),  \"metric\"",
+          "query": "SELECT mean(\"value\") FROM \"wordcount_py27_results\" WHERE metric = 'wordcount_it_runtime' AND $timeFilter GROUP BY time($__interval),  \"metric\"",
```

Comment:
Seeing this query now - yes, I wound just keep the metric 'runtime', since we already know it is wordcount_py27_results, and it would be simpler that pipeline does not need to know the name of the suite. In the future we might add different metrics like 'cost' or total cputime consumed by other workers as opposed to runtime.

Reply:
I agree, `runtime` will be enough.

Reply:
I changed it to 'runtime'